examples/favorites_advanced/bloc_favorites_advanced/lib/feature_puppy/search/views/search_page.dart

```diff
@@ -44,28 +42,26 @@ class SearchPage extends StatelessWidget {
               itemCount: state.searchedPuppies!.length,
               itemBuilder: (context, index) {
                 final item = state.searchedPuppies![index];
-                // print('PuppyListStatus  : ${state.status}');
                 return PuppyCard(
                   key: Key('${Keys.puppyCardNamePrefix}${item.id}'),
                   onVisible: (puppy) => context
                       .read<PuppiesExtraDetailsBloc>()
                       .add(FetchPuppyExtraDetailsEvent(puppy)),
                   puppy: item,
-                  onCardPressed: (puppy) {},
+                  onCardPressed: (puppy) {
```

Comment:
@StanislavPrime, here you can use a body expression

Reply:
I converted the function to an expression-bodied function: 
`onCardPressed: (puppy) =>
                    Navigator.of(context).push(PuppyFlow.route(puppy: puppy))`